go.mod

```diff
@@ -3,19 +3,24 @@ module github.com/giantswarm/certctl
 go 1.13
 
 require (
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/giantswarm/apprclient v0.2.0
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/e2e-harness v0.2.0
-	github.com/giantswarm/e2etemplates v0.2.0
 	github.com/giantswarm/go-uuid v0.0.0-20141202165402-ed3ca8a15a93
-	github.com/giantswarm/helmclient v0.2.0
+	github.com/giantswarm/helmclient v1.0.0
 	github.com/giantswarm/k8sclient v0.2.0
 	github.com/giantswarm/microerror v0.2.0
 	github.com/giantswarm/micrologger v0.3.1
 	github.com/giantswarm/vaultrole v0.2.0
 	github.com/hashicorp/vault/api v1.0.4
 	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53
+	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.5
-	k8s.io/apimachinery v0.16.6
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/helm v2.16.4+incompatible
```

Comment:
Can we drop this too? 

Maybe we're using the helm options like `helm.ReleaseName` somewhere? These are now part of helmclient.

Reply:
Good catch, let me search for that usage.

Reply:
I found out I also need to remove `replace` section in go.mod. Thanks for notice!